src/WattleScript.Interpreter/Tree/Statements/MixinDefinitionStatement.cs

```diff
@@ -138,7 +138,7 @@ public override void Compile(FunctionBuilder bc)
                 bc.Emit_Literal(DynValue.NewString(field.Name));
             }
             if (privateCount > 0) {
-                bc.Emit_SetPriv(privateCount);
+                bc.Emit_SetFlags(privateCount, MemberModifierFlags.Private);
```

Comment:
Same: Complete with list of public fields

Reply:
fixed in 6f5e0fe